Import credential-management WPT tests

maraisr opened the pull request:
#### 0ba313f0755d90540c9c97a08e481c192f78c295
<pre>
Import credential-management WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=282820">https://bugs.webkit.org/show_bug.cgi?id=282820</a>

Reviewed by Anne van Kesteren.

Import of wpt credential-management tests, and removed duplicate tests
in http/wpt.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/c6e7febe9c5f8cca44c4e6e110104b4e12097fdf">https://github.com/web-platform-tests/wpt/commit/c6e7febe9c5f8cca44c4e6e110104b4e12097fdf</a>

* LayoutTests/http/wpt/credential-management/credentialscontainer-create-basics.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-create-basics.https.html: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-get-basics.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-get-basics.https.html: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-preventSilentAccess-basics.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-preventSilentAccess-basics.https.html: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-store-basics.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-store-basics.https.html: Removed.
* LayoutTests/http/wpt/credential-management/idl.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/idl.https.html: Removed.
* LayoutTests/http/wpt/credential-management/setDigitalCreditialsEnable.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-create-basics.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-create-basics.https.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-frame-basics.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-frame-basics.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-get-basics.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-get-basics.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-prevent-silent-access.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-prevent-silent-access.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/non-fully-active.https.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-get-basics.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-get-basics.https.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-iframe.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-iframe.https.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-store.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-store.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/support/README.md:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/support/otpcredential-helper.js:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/support/otpcredential-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/286403@main">https://commits.webkit.org/286403@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ce4d13eef130a9a3bac9a0c9af941b7df96b70c8

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/75882 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/54911 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/28779 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/80379 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/27148 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/64053 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/3205 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/59497 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/17658 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/78949 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/49374 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/65170 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/39857 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/46774 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/22653 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/25475 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/67889 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/22991 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/81843 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/3251 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/2047 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/67726 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/3405 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/65138 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/67033 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/10977 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/9104 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/11733 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/3201 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/3222 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/4160 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/3229 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (approved):
Assuming the bots are okay, this looks good to me. Thanks for cleaning it up!

cc @pascoej

maraisr committed:
Import credential-management WPT tests
https://bugs.webkit.org/show_bug.cgi?id=282820

Reviewed by Anne van Kesteren.

Import of wpt credential-management tests, and removed duplicate tests
in http/wpt.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/c6e7febe9c5f8cca44c4e6e110104b4e12097fdf

* LayoutTests/http/wpt/credential-management/credentialscontainer-create-basics.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-create-basics.https.html: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-get-basics.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-get-basics.https.html: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-preventSilentAccess-basics.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-preventSilentAccess-basics.https.html: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-store-basics.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/credentialscontainer-store-basics.https.html: Removed.
* LayoutTests/http/wpt/credential-management/idl.https-expected.txt: Removed.
* LayoutTests/http/wpt/credential-management/idl.https.html: Removed.
* LayoutTests/http/wpt/credential-management/setDigitalCreditialsEnable.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-create-basics.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-create-basics.https.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-frame-basics.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-frame-basics.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-get-basics.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-get-basics.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-prevent-silent-access.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/credentialscontainer-prevent-silent-access.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/non-fully-active.https.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-get-basics.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-get-basics.https.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-iframe.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-iframe.https.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-store.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/otpcredential-store.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/credential-management/support/README.md:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/support/otpcredential-helper.js:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/support/otpcredential-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/credential-management/w3c-import.log:

Canonical link: https://commits.webkit.org/286403@main
